app/src/migration/java/org/mozilla/fenix/MigratingFenixApplication.kt

```diff
@@ -31,7 +33,7 @@ class MigratingFenixApplication : FenixApplication() {
                 this.components.core.pinnedSiteStorage
             )
             .migrateLogins(this.components.core.lazyPasswordsStorage)
-            .migrateFxa(lazy { this.components.backgroundServices.accountManager })
+            .migrateFxa(lazy { this.components.backgroundServices.accountManager }, fxaExpectChinaServers)
```

Comment:
Was this flag present in the migration builds you already shipped? I ask because if it wasn't present, Mozilla-Online users who were not using Global server were logged-out... Hopefully that didn't happen.

Reply:
Yes, the builds we shipped have already had this flag present.